addon/components/bs-form.js

```diff
@@ -154,18 +155,6 @@ export default Ember.Component.extend({
   novalidate: false,
 
   /**
-   * An array of `Components.FormElement`s that are children of this form.
-   *
-   * @property childFormElements
-   * @type Array
-   * @readonly
-   * @protected
-   */
-  childFormElements: computed.filter('childViews', function(view) {
```

Comment:
I would prefer to keep the `childFormElements` in place, but just redefine it to operate on the `children` property instead of on `childViews`. This way we make sure we only operate on FormElement instances, if there is some weird case where we have another child component having the child mixin that is not a FormElement! And maybe some users have subclassed bs-form and used the `childFormElements` property, this way we would not break anything...